src/release_notes_workflow/releasenotes_gitlog_getcommit.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import subprocess
+
+
+class GitLogGetCommit:
+
+    @classmethod
+    def gitlog_getcommitID(self, gitrepo: str, date: str) -> str:
+        gitLogCmd = f'git log --date=local --after={date} --pretty=format:"%h"'
+        gitLogCommitID = subprocess.check_output(gitLogCmd.split(), cwd=gitrepo).decode()
+        return gitLogCommitID
+
+    @classmethod
+    def gitlog_getdate(self, gitrepo: str, date: str) -> str:
```

Comment:
This method is identical to the above minus the format. Implement a method called `commits` that returns a collection of instances of a new class called `GitCommit`, each having properties like `id`. Note that if you use a git library, that's likely already implemented for you. 

When you want the date of the last commit or the id the code should look like so:

```
last_commit = next(repo.commits(after))
# you have last_commit.date and lastcommit.id
```

Reply:
The code should now support  `.commitID` `.commitDate`
https://github.com/opensearch-project/opensearch-build/blob/8e23f2b7da606024053504c1ace23c53b488fbed/src/release_notes_workflow/releasenotes_gitlog_getcommit.py